charts/sysdig-stackdriver-bridge/README.md

```diff
@@ -2,61 +2,81 @@
 
 [Sysdig Stackdriver Bridge](https://docs.sysdig.com/en/kubernetes-audit-logging.html#UUID-f62c275e-389a-317f-2079-2c61d1f282a7_UUID-ded20060-405c-1f5f-4b3f-c18d20b5668d) is an extension of the Sysdig tool that reads audit logs from Stackdriver, reformats them to match the Kubernetes-native format, and sends the logs to a configurable webhook and to the Sysdig agent service.
 
-## Introduction
 
 This chart adds the Sysdig Stackdriver Bridge application into a cluster using a simple `Deployment` with a `ConfigMap`.
 
 ## Prerequisites
 
-- Sysdig should be deployed in the target Kubernetes cluster.
-- You will need to create a secret that matches the value `secret.name` (`stackdriver-webhook-bridge` by default) that contains the key `key.json` with the value of the created GCP IAM Service Account for Sysdig to access Stackbridge.
-- It is recommended to create this `Secret` in an out-of-band fashion in CI/CD, etc.
-  - Optionally, you can enable creation of the `Secret` via the chart by setting `secret.create=true` and providing the JSON data directly via `secret.data`. **This is not an advisable approach for production systems.**
+- Sysdig is deployed in the target Kubernetes cluster.
+- A secret that matches the value `secret.name` (`stackdriver-webhook-bridge` by default) is created. It should contain the `key.json` key with the value of the created GCP IAM Service Account for Sysdig to access Stackbridge.
+  - It is recommended to create this `Secret` in an out-of-band fashion in CI/CD, etc.
+  - Optionally, you can enable the creation of the `Secret` via the chart by setting `secret.create=true` and providing the JSON data directly via `secret.data`. **This is not an advisable approach for production systems.**
 
-## Installing the Chart
+## Install the Chart
 
-To install the chart with the release name `my-release`, run:
+To install the chart with the release name `my-release`:
 
-```bash
-$ helm repo add sysdiglabs https://sysdiglabs.github.io/charts/
-```
+1. Add the `sysdiglabs` Helm chart repository. 
```

Comment:
@radhikapc it seems there's an extra space at the end of this line. Can u please remove it? 🙏🏻

Reply:
done. please see